anomalib/data/__init__.py

```diff
@@ -14,6 +14,7 @@
 from .folder import Folder
 from .inference import InferenceDataset
 from .mvtec import MVTec
+from .task_type import TaskType
```

Comment:
would this be used by models and other components as well? If so, would it be an idea to move it to a place that would be accessible by other components?

Reply:
Yeah, I'm not super happy with the location of `TaskType` but couldn't think of a better place. Maybe somewhere in `anomalib/config` would work? @ashwinvaidya17 any ideas?